orbiting/haku-frontend
# clean up edit page & misc

Changes
- redirect to latest commit if none provided and repo exists
- fix local storage warning
- update route & store after commit (ensure field consistency with query fragment)
- always save to correct commit id in local storage
- single code path for `uncommittedChangesMutation`
- clean up
- use `repos` and `me.roles` from api
- more consistent naming with backend
  - new routes

Todo
- [x] completely translate sidebar
- [x] better error display
- [x] test & enable article creation (depends on backend)
